Mlem/App/Views/Shared/Search/SearchView+Logic.swift

```diff
@@ -102,6 +105,34 @@ extension SearchView {
         try await postLoader.refresh(clearBeforeRefresh: clearBeforeRefresh)
     }
     
+    public func refreshComments(clearBeforeRefresh: Bool) async throws {
+        guard !query.isEmpty else { return }
+        await commentLoader.searchCommentFetcher.changeApi(
+            to: getRefreshApi(for: commentFilters.location)
```

Comment:
```suggestion
            to: getRefreshApi(for: commentFilters.location, context: filtersTracker.filterContext)
```

Compilation error on this line due to middleware changes

Reply:
It builds for me. Did you pull https://github.com/mlemgroup/mlem/pull/1626/commits/316ed6e4b5ae9584fb6357d126b38543fc8e9f24?

Reply:
Yeah, looks like it was a package issue--cleared the caches and it built fine.